server/src/server.ts

```diff
@@ -405,6 +405,10 @@ connection.onDefinition((textDocumentPosition: TextDocumentPositionParams): Loca
         return Location.create(uri, range);
     }
 
+    function makeDataUri(doc: TextDocumentPositionParams, path: string): string {
+        return 'gltf-dataUri:' + path + '#' + encodeURIComponent(Uri.parse(doc.textDocument.uri).fsPath);
```

Comment:
Out of curiosity, why did the `//x/` part go away?  I guess we don't need the protocol/authority separator here anymore?

Reply:
On Linux the previous path would be something like //user where on Windows it would be /C:\bla. I Added the 'x' because the uri would without be something like gltf-dataUri:////user on Linux and that was causing an error in the URI parser.  With this we just have a single '/' gltf-dataUri:/accessor and its legal.